stream_alert/rule_processor/handler.py

```diff
@@ -25,21 +28,26 @@
 from stream_alert.shared.metrics import MetricLogger
 
 
+MAX_RECORD_SIZE = 1000 * 1000
+MAX_BATCH_SIZE = 500
+
+
 class StreamAlert(object):
-    """Wrapper class for handling all StreamAlert classificaiton and processing"""
+    """Fascade for handling StreamAlert classificaiton and processing"""
```

Comment:
Why `Fascade`?

Reply:
Because the class exists to include other classes

Reply:
but fascade isn't a word? And also not a common term, while wrapper is both common and explicit